backend/src/schemas/slide-object.schema.ts

```diff
@@ -176,10 +179,10 @@ export class SlideObject {
   skewY: number;
 
   @Prop({
-    type: [[String, Number, Number]],
+    type: Array<SlideObjectPath>,
```

Comment:
We need to decide on a single convention - either 
```
Array<sth>
```
or
```
sth[]
```
Second option is shorter and more consise imo

Reply:
Changed to `[]` where I could, but in `@Prop` it must be `Array`:

![image](https://github.com/user-attachments/assets/9dae3dbf-ab7b-4a7c-9236-2fcdcf04cb88)

Reply:
```
  @Prop({
    type: [
      {
        type: MongooseSchema.Types.Mixed,
        properties: {
          type: {
            type: String,
            required: true,
          },
          value: {
            type: Number,
            required: true,
          },
        },
      },
    ],
  })
  filters?: SlideObjectFilter[];
  ```

Reply:
i don't like this solution, because if something changes it the interface, you have to fix it in two spots.

Reply:
Also if we want to apply it to `SlideObjectTextStyles` it would get very complicated

Reply:
properties are optional, you dont need to use them

Reply:
so just `[ { type: MongooseSchema.Types.Mixed}]` for all of them?

Reply:
For now its enough imo

Reply:
Done